### PR TITLE
LL-2136 Account ordering prioritize starred accounts + tests

### PR DESCRIPTION
**Pull request by juan-cortes**
Allows to sort accounts taking into account starred accounts by receiving an id array. We will need to update the call to `sortAccountsComparatorFromOrder` on lld and eventually llm when we have stars over there too.

### CODE REVIEW

File: src/account/ordering.js [hunk 1]
```diff
@@ -29,13 +29,20 @@ export const sortAccountsComparatorFromOrder = (
   calculateCountervalue: (
     currency: TokenCurrency | CryptoCurrency,
     value: BigNumber
-  ) => ?BigNumber
+  ) => ?BigNumber,
+  starredAccountIds: string[]
```
Comment (gre) [inline src/account/ordering.js]: can you make it so it's backward compatible? third param to fallbacks to `[]`

Comment (juan-cortes) [reply src/account/ordering.js]: Makes sense, yeah I can do that

Comment (juan-cortes) [reply src/account/ordering.js]: Done, with test 🚀

File: src/__tests__/accounts/ordering.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+      "libcore:1:ethereum:xpub3:",
+      "libcore:1:ethereum:xpub3B:",
+      "libcore:1:ethereum:xpub1B:"
+    ]
+  );
+  const sortedAccounts = accounts.sort(compareFn);
+  expect(sortedAccounts.map(a => a.name)).toEqual(["AA", "C", "CA", "B", "A"]);
+});
+
+test("Accounts ordering | backwards compatible when no starred accounts are passed", () => {
```
Comment (gre) [inline src/__tests__/accounts/ordering.js]: Thanks for doing this test too :rocket:

**Pull request merged**